wamr-sdk/app/libc-builtin-sysroot/include/stdint.h

```diff
@@ -11,6 +11,9 @@ extern "C" {
 #endif
 
 /* clang-format off */
+/* The word size of platform */
+#define __WORDSIZE 32
```

Comment:
Though currently wasm in spec mvp is 32-bit, but wasm has introduced the [memory64](https://github.com/WebAssembly/memory64) spec proposal, had better not assume wordsize is 32-bit here. Is there a better way to control the code?

Reply:
Thank you very much. I got it!

Reply:
I use macros like `__wasm32__` to identify `wasm32 or wasm64`. This macro is predefined by `clang`

Reply:
OK, can you try both wasi-sdk and emsdk to check whether they both define the macro?

Reply:
Ok, I got it.

Reply:
I haven't used `emsdk` before. I used the following commands and programs to check and got the following output. It seems that emsdk also uses clang. And the macro ` __wasm32__` is valid.
* command
```shell
emcc ./main.c -o main
```
* source code
```c
#include<stdio.h>
#ifdef __wasm32__
#error WASM32 detected!
void wasm32(){
    printf("wasm32");
}
#endif
#ifdef __wasm64__
#error wasm64
void wasm64(){
    printf("wasm64");
}
#endif
int main(){
#ifdef __wasm32__
wasm32();
#endif
#ifdef __wasm64__
wasm64();
#endif
    return 0;
}
```
* result
```
./main.c:3:2: error: WASM32 detected!
#error WASM32 detected!
 ^
1 error generated.
emcc: error: '/home/ljq/test/emsdk/upstream/bin/clang -target wasm32-unknown-emscripten -fignore-exceptions -fvisibility=default -mllvm -combiner-global-alias-analysis=false -mllvm -enable-emscripten-sjlj -mllvm -disable-lsr -DEMSCRIPTEN -Werror=implicit-function-declaration --sysroot=/home/ljq/test/emsdk/upstream/emscripten/cache/sysroot -Xclang -iwithsysroot/include/fakesdl -Xclang -iwithsysroot/include/compat ./main.c -c -o /tmp/emscripten_temp_amzvebaf/main_0.o' failed (returned 1)
```

Reply:
Got it, thanks.